indigo/distribution.yaml

```diff
@@ -4221,6 +4221,12 @@ repositories:
       url: https://github.com/kf/kingfisher.git
       version: indigo-devel
     status: maintained
+  kml_util:
+    doc:
+      type: git
+      url: https://github.com/silviomaeta/kml_util.git
+      version: indigo-devel
```

Comment:
Yes.  You are right.   It is master.  I will fix it and submit the request again. 
Thanks,
Silvio

Dirk Thomas <notifications@github.com> wrote:

>> @@ -4221,6 +4221,12 @@ repositories:
>>        url: https://github.com/kf/kingfisher.git
>>        version: indigo-devel
>>      status: maintained
>> +  kml_util:
>> +    doc:
>> +      type: git
>> +      url: https://github.com/silviomaeta/kml_util.git
>> +      version: indigo-devel
>
>There is no such branch name in your repo. This should probably be `master`?
>
>Same below.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/ros/rosdistro/pull/10710/files#r55925126